### ISSUE TITLE
Notifications blinking when displaying long text with picom

### ISSUE THREAD

**Issue by VerdeQuar** [Toqozz/wired-notify#45]
Notifications are blinking when displaying long text with picom running
Sometimes when just one notification is on screen it doesn't happen but since every notification is a new window it shoudn't make a diffrence so I spammed them so it's easier to see

I'm using default configuration with wired from AUR, and [here](https://pastebin.com/G1huXfcZ) is my picom.conf

Picom running, short text: https://i.imgur.com/zLNpE9V.gif
![zLNpE9V](https://user-images.githubusercontent.com/35724192/147885992-3845d8fd-06ec-4a1b-8f84-29ab1a1355c5.gif)

Picom running, long text: https://i.imgur.com/spjtWmX.gif
![spjtWmX](https://user-images.githubusercontent.com/35724192/147886014-e5dd6803-50ed-405f-a08e-e97c7de72730.gif)

Picom killed, short text: https://i.imgur.com/b87TrEN.gif
![b87TrEN](https://user-images.githubusercontent.com/35724192/147886034-47c37ceb-2210-49c8-a729-11402e401aa4.gif)

Picom killed, long text: https://i.imgur.com/01MzJS8.gif
![01MzJS8](https://user-images.githubusercontent.com/35724192/147886053-98a585bd-2852-433b-83db-80f9ed6869fe.gif)

**Comment by Toqozz** [comment]
Hmm... I can't reproduce this on my machine.

I tried your picom config but I had to use `blur-method = "kernel"` instead.

One issue revealed here in the gif is that an in-between state is being drawn which looks ugly as heck:
![img-2022-01-03-103548](https://user-images.githubusercontent.com/6947819/147893978-d1cf1b45-739a-4998-8dce-8f4d46ec785b.png)

This should be fixed in the latest master.  I'm not sure it will fix your issue, would you care to try it?

Otherwise, I have a couple questions:
- Does it happen more when there are more notifications on screen?
- Does it happen more/less if you change `poll_interval` to be lower or higher? (in `wired.ron`)

**Comment by VerdeQuar** [comment]
Ok so I did try rebooting earlier to see if it would go away and it didn't but today it's gone so I guess there is no problem anymore
I did try wired-git from AUR but it didn't compile, didn't bother with compiling it directly from github
And yeah it seemed that when there was more notifications on screen it was worst but cause it magically fixed itself I have no way of confirming this

**Comment by VerdeQuar** [comment]
So I tried wired-git from AUR but it didn't work at all it compiled correctly but no notifications are shown, didn't bother with compiling it directly from github
It doesn't seem to make a difference how many notifications there are
but poll_interval definitely makes a difference: the lower poll_interval is the quicker it blinks

**Comment by Toqozz** [comment]
`wired-git` is definitely working for me, and I haven't heard about any other issues with it.

Please note that the config was updated recently.  If you had copied the old default config and were using that then it won't work anymore (unless there is an image hint present) because of some changes in render criteria.

**Comment by Toqozz** [comment]
Right, I think I may know what that could be and will have a look.

**Comment by Toqozz** [comment]
I believe the transparency issue should be fixed now, but let me know if you have more issues because transparency in general is just a bit weird with cairo and xlib.

**Comment by VerdeQuar** [comment]
It's working now, thanks
